src/MongoDB/Driver/Manager.cpp

```diff
@@ -563,6 +728,7 @@ void HHVM_METHOD(MongoDBDriverManager, __construct, const String &dsn, const Arr
 	data->m_client = client;
 
 	hippo_mongo_driver_manager_apply_ssl_opts(data->m_client, driverOptions);
+	hippo_mongo_driver_manager_set_monitoring_callbacks(data->m_client, this_);
```

Comment:
Similar to what I asked in https://github.com/mongodb/mongo-php-driver/pull/466#discussion_r108456224, doesn't it only make sense to do this when creating a new `mongoc_client_t` and not re-using a persisted client?

The same applies for calling `hippo_mongo_driver_manager_apply_ssl_opts()` on the previous line (although that wasn't introduced in this PR). PHPC definitely does not apply SSL opts on subsequent requests.